base/util.go

```diff
@@ -590,16 +591,40 @@ func IsFilePathWritable(fp string) (bool, error) {
 	return true, nil
 }
 
-// Replaces sensitive data from the URL query string with ******.
+// SanitizeRequestURL will return a sanitised string of the URL by:
+// - Tagging mux path variables.
+// - Replacing sensitive data from the URL query string with ******.
 // Have to use string replacement instead of writing directly to the Values URL object, as only the URL's raw query is mutable.
-func SanitizeRequestURL(requestURL *url.URL) string {
-	urlString := requestURL.String()
+func SanitizeRequestURL(rq *http.Request) string {
+	urlString := SanitizeRequestURLQueryParams(rq.URL)
+
+	// Apply tagging to path variables
+	pathVars := mux.Vars(rq)
+	if (RedactSystemData || RedactMetadata || RedactUserData) && len(pathVars) > 0 {
+		for k, v := range pathVars {
+			if strings.HasPrefix(k, "UD") {
```

Comment:
Now I've switched to using the `UD_` notation for path variable names, I can add that in to reduce false positives.